sys/linux/socket_netlink_route.txt

```diff
@@ -879,6 +877,32 @@ rtcanmsg {
 
 rtcanmsg_flags = CGW_FLAGS_CAN_ECHO, CGW_FLAGS_CAN_SRC_TSTAMP, CGW_FLAGS_CAN_IIF_TX_OK
 
+lwtunnel_encap_attr [
+	generic	array[int8]
+	typed	nlattr_tt[int16:14[0:150], 0, 0, lwtunnel_encap_data]
+	nested	nlattr_tt[int16:14[0:150], 0, 1, array[lwtunnel_encap_data_nonested]]
+] [varlen]
+
+lwtunnel_encap_data_nonested [
+	generic		array[int8]
+	ipv6_rpl_encap	nlattr_tt[int16:14[0:150], 0, 0, lwtunnel_encap_data]
+] [varlen]
+
+lwtunnel_encap_data [
+#TODO: define remaining lwtunnel_encap types
+	generic		nl_generic_attr_data
+	ipv6_rpl_encap	ipv6_rpl_sr_hdr
+] [varlen]
+
+ipv6_rpl_sr_hdr {
+	nexthdr		int8
+	hdrlen		int8
+	type		int8
```

Comment:
I resolved this by adding `len` type: `len[parent, int8]`, and `const` type: `const[RPL_IPTUNNEL_SRH, int8]`, instead of using `int8` type for both.

PTAL